src/main/resources/feedbackQuestionTextSubmissionFormTemplate.html

```diff
@@ -1,7 +1,5 @@
 <div class="panel panel-default panel-body"
-    ${disabled} id="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
-    name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
-    data-length-text-id="responseLength-${questionIndex}-${responseIndex}" data-recommended-text="${recommendedLength}">
+    data-is-session-closed="${isSessionClosed}" id="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}" name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}" data-length-text-id="responseLength-${questionIndex}-${responseIndex}" data-recommended-text="${recommendedLength}">
```

Comment:
@amarlearning Apologies if I wasn't clear. When I said "ignore the above comment" I meant the one about not changing the order of attributes in case you have to rerun god mode. My [first comment](https://github.com/TEAMMATES/teammates/pull/7811#discussion_r129121116) about the unrelated change of putting all attributes on the same line still holds. That is the only change that is pending.